hack/build-manifests.sh

```diff
@@ -28,13 +28,16 @@ kubevirt_logo_path="assets/kubevirt_logo.png"
 rm -rf ${MANIFESTS_OUT_DIR}
 rm -rf ${MANIFEST_TEMPLATES_OUT_DIR}
 
-rm -rf "${TESTS_OUT_DIR}/tools"
-mkdir -p "${TESTS_OUT_DIR}/tools"
-templator=${TESTS_OUT_DIR}/tools/manifest-templator
-
-bazel run \
-    --config=${HOST_ARCHITECTURE} \
-    //:build-manifest-templator -- ${templator}
+templator=${MANIFEST_TEMPLATOR}
```

Comment:
A clear indicator if one wants to use `bazel` or `go` would be something I would prefer. It a little bit a mix-and-match with unclear fallback paths for some users. Could you improve that a little?

Reply:
I was thinking about moving the build of `manifest-templator` tool to a separate make target. Or even better: include it in the `bazel-build` / `go-build`. That way `hack/build-manifests.sh` can only do the manifests generation. What do you think? Is there any reason to keep `bazel run ... //:build-manifest-templator ...` here?